include/oneapi/dpl/pstl/hetero/algorithm_impl_hetero.h

```diff
@@ -1621,6 +1625,8 @@ __pattern_hetero_set_op(_ExecutionPolicy&& __exec, _ForwardIterator1 __first1, _
     using _MaskAssigner = unseq_backend::__mask_assigner<2>;
     using _InitType = unseq_backend::__no_init_value<_Size1>;
     using _DataAcc = unseq_backend::walk_n<_ExecutionPolicy, oneapi::dpl::__internal::__no_op>;
+    using _AlgoType = ::std::integral_constant<::oneapi::dpl::__internal::__algorithm_type,
+                                               ::oneapi::dpl::__internal::__algorithm_type::set_operation>;
```

Comment:
It seems like this and similar instances would benefit from a unified helper to shorten it to require only the second template argument.